crates/api/src/op_store.rs

```diff
@@ -88,7 +88,6 @@ pub trait OpStore: 'static + Send + Sync + std::fmt::Debug {
         arc: DhtArc,
         start: Timestamp,
         end: Timestamp,
-        limit_bytes: Option<u32>,
```

Comment:
Oh and bonus, we wanted these two out of the API for the op store, so 50% done :)